packages/palette/src/elements/Typography/Typography.tsx

```diff
@@ -186,6 +198,7 @@ function createStyledText<P extends StyledTextProps>(
             fontFamilyType && themeProps.fontFamily[fontType][fontFamilyType]
           }
           {...determineFontSizes(fontType, size)}
+          {...(unstable_trackIn ? { letterSpacing: "-0.03em" } : {})}
```

Comment:
So I guess ideally this should be passed to `determineFontSizes` and only applied if there is a proper set up for it I think. But for now applying it directly here.